packages/form/sandbox/index.tsx

```diff
@@ -0,0 +1,264 @@
+import { AtomState, createCtx } from "@reatom/core";
+import { connectLogger } from "@reatom/logger"
+import { reatomContext, useAction, useAtom } from "@reatom/npm-react"
+import { FieldAtom, FormFieldArrayAtom, reatomForm, withField } from "../src";
+import { LinkedListAtom, LinkedListLikeAtom, reatomBoolean } from "@reatom/primitives";
+import { PropsWithChildren } from 'react';
+import { createRoot } from "react-dom/client";
+import { useFormField } from "./use-form-field";
+
+const ctx = createCtx();
+connectLogger(ctx);
+
+function App() {
+	return (
+		<reatomContext.Provider value={ctx}>
+			<Form />
+		</reatomContext.Provider>
+	)
+}
+
+const root = createRoot(document.getElementById('root')!)
+root.render(<App />)
+
+const form = reatomForm(fieldArray => ({
+	username: 'vlad',
+	addresses: [
+		{
+			country: 'my country',
+			street: 'my street',
+			city: 'my city',
+			tags: ['defaultTag', 'defaultTag2'],
+			phoneNumbers: fieldArray({
+				initState: Array<{ number: string, priority: boolean }>(),
+				create: (ctx, { number, priority }) => ({
+					number,
+					priority: reatomBoolean(priority).pipe(withField(priority))
+				})
+			})
```

Comment:
I didn't agree with "fieldArray" concept. It should be just `phoneNumbers: Array<{ number: string, priority: boolean }>()` which we use to create LinkedList WITH our create function which accept and parse JSON just like the reatomForm init state. And if a user need a very custom logic they should write `phoneNumbers: reatomLinkedList(...)` or `phoneNumbers: reatomSet(...)` and so on.

Reply:
Ok, lets do it as a separate exported method and will play with that